admin_ui/src/views/Login.vue

```diff
@@ -68,8 +63,11 @@ export default {
             } else {
                 this.$router.push({ name: "home" })
             }
-        },
+        }
     },
+    updated() {
+        this.$i18n.locale = this.defaultLanguage
+    }
```

Comment:
Do we need this? Can we do it all in App.vue.

``updated`` gets called every time the DOM updates.

Reply:
Unfortunately, from `created()` in `App.vue` doesn't work. It only works from `Login.vue` (when admin is login to UI `default_language` is set). I'll change to `mounted()` lifecycle hook.

Reply:
I left the` updated()` hook because if I use `mounted()` and, for example, refresh the login page, the default language is not applied in the UI. If you have any other idea, feel free to change it.